sdcm/nemesis.py

```diff
@@ -1548,6 +1548,66 @@ def disrupt_network_block(self):
         finally:
             self.target_node.traffic_control(None)
 
+    def disrupt_cluster_down_scaling(self):
+        """
+        https://docs.scylladb.com/operating-scylla/procedures/cluster-management/remove_node/
+
+        1.terminate
+        2.Full repair
+        3.Nodetool removenode
+        4.Sleep 10 minutes
+        5.Add new node
+        """
+
+        # pick random node
+        self._set_current_disruption('ScaleDown')
```

Comment:
the name here should be same as the class, i.e. this is what would be shown in grafana,
I would rather it be `Removenode`

Reply:
done